metrics-server/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Build stage
+FROM quay.io/cybozu/golang:1.13-bionic AS build
+
+ARG METRICS_SERVER_VERSION=0.3.6
+ARG CGO_ENABLED=0
+WORKDIR /go/src/github.com/kubernetes-incubator
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN git clone --branch v${METRICS_SERVER_VERSION} https://github.com/kubernetes-sigs/metrics-server.git && \
```

Comment:
Please download a tarball from the release page.
It is lighter than `git clone`.
https://github.com/kubernetes-sigs/metrics-server/archive/v0.3.6.tar.gz